gitpod-io/website
# Document `gp ports list` command

## Description

Document the `gp ports list` command.

## Related Issue(s)

Resolves #2171

## How to test


Confirm if the following pages* mention the `gp ports list` command:
- [docs/config-ports](https://deploy-preview-2200--gitpod-kumquat.netlify.app/docs/config-ports)
- [docs/command-line-interface](https://deploy-preview-2200--gitpod-kumquat.netlify.app/docs/command-line-interface)

_*These pages are from the preview environment of this PR._

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2200"><img src="https://gitpod-staging.com/button/open-in-gitpod.svg"/></a>



<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/2200"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
